io.gitlab.celleron56.libretile.yaml

```
@@ -0,0 +1,96 @@
+app-id: io.gitlab.celleron56.libretile
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: start.sh
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --socket=pulseaudio
+  - --device=all
+  - --share=network
+
+
+modules:
+
+  - name: tkinter
+    buildsystem: simple
+    build-commands:
+      - pip3 install --prefix=${FLATPAK_DEST} .
+    sources:
+      - type: git
+        url: https://github.com/iwalton3/tkinter-standalone
+        commit: ba946536054f9d27a08aafde21aa18330ce05729
+    modules:
+      - name: tcl
+        buildsystem: autotools
+        subdir: unix
+        post-install:
+          - chmod 755 /app/lib/libtcl*.so
+        cleanup:
+          - /bin
+          - /lib/pkgconfig
+          - /man
+        sources:
+          - type: archive
+            url: https://prdownloads.sourceforge.net/tcl/tcl8.6.12-src.tar.gz
+            sha256: 26c995dd0f167e48b11961d891ee555f680c175f7173ff8cb829f4ebcde4c1a6
+            x-checker-data:
+              type: anitya
+              project-id: 4941
+              stable-only: true
+              url-template: https://prdownloads.sourceforge.net/tcl/tcl$version-src.tar.gz
+      - name: tk
+        buildsystem: autotools
+        subdir: unix
+        post-install:
+          - chmod 755 /app/lib/libtk*.so
+        cleanup:
+          - /bin
+          - /lib/pkgconfig
+          - /man
+        sources:
+          - type: archive
+            url: https://prdownloads.sourceforge.net/tcl/tk8.6.12-src.tar.gz
+            sha256: 12395c1f3fcb6bed2938689f797ea3cdf41ed5cb6c4766eec8ac949560310630
+            x-checker-data:
+              type: anitya
+              project-id: 11426
+              stable-only: true
+              url-template: https://prdownloads.sourceforge.net/tcl/tk$version-src.tar.gz
+
+      - name: python3-pillow
+        buildsystem: simple
+        build-commands:
+         - pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}"
+           --prefix=${FLATPAK_DEST} "pillow==9.2.0" --no-build-isolation
```

```suggestion
           --prefix=${FLATPAK_DEST} "Pillow" --no-build-isolation
```

This is less burden when you update the version. There is only one "Pillow" to install.